src/lib/getMessageManager.ts

```
@@ -2,6 +2,7 @@
 import * as Clipboard from 'expo-clipboard'
 import once from 'lodash.once'
 import { injectedJavaScript as injectedJavaScriptClipboard } from './clipboard'
+import * as StoreReview from 'expo-store-review';
```

ESLint

[error] Delete `;`

(prettier/prettier)